File: Euclid/Messages/Outgoing/REMOVEITEM.cs
using Euclid.Game;
using Euclid.Network.Streams.Util;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Euclid.Messages.Outgoing
{
    class REMOVEITEM : IMessageComposer
    {
        private Item item;

        public REMOVEITEM(Item item)
        {
            this.item = item;
        }

        public override void Write()
        {
            Data.Add(new ArgumentEntry(this.item.Data.Id));
        }
    }
}

File: euclid/src/main/java/euclid/messages/outgoing/REMOVEITEM.java
package euclid.messages.outgoing;

import euclid.game.item.Item;
import euclid.messages.IMessageComposer;
import euclid.network.streams.util.ArgumentEntry;

public class REMOVEITEM extends IMessageComposer {

    private final Item item;

    public REMOVEITEM(Item item) {
        this.item = item;
    }

    @Override
    public void write() {
        getData().add(new ArgumentEntry(item.getData().getId()));
    }
}
